tests/test_whitelist.py

```diff
@@ -9,12 +13,9 @@ def test_read_whitelist(mock_db):
     """
     whitelist = ModuleFactory().create_whitelist_obj(mock_db)
     mock_db.get_whitelist.return_value = {}
-    (
-        whitelisted_IPs,
-        whitelisted_domains,
-        whitelisted_orgs,
-        whitelisted_mac,
-    ) = whitelist.read_whitelist()
+    whitelisted_IPs, whitelisted_domains, whitelisted_orgs, whitelisted_mac = (
+        whitelist.read_whitelist()
+    )
```

Comment:
i think we can use mock file for testing the read_whitelist function, this is an important one so we probably should have more strict tests for it. for example 2  whitelist test files would be enough, and in these files we can check for expected values, missing columns, missing values in a line, etc. what do you think @Sekhar-Kumar-Dash

Reply:
yes you are right